### DISCUSSION TITLE
WebSocket connection issue

### DISCUSSION THREAD

**Discussion by jarojetu** [ExperienceLovelace/ha-floorplan#425] [Q&A] [answered]
Hi, I would like to ask someone for help. Im running my HA on docker and trying to use ha-floorplan. After installation and looking though the videos and docs I prepared my sample test but getting the errors inside the console. The dashboard is completely empty and showing nothing.
![image](https://github.com/ExperienceLovelace/ha-floorplan/assets/17701406/7e0a56ae-27d5-4c5f-98c3-21667551e1f8)

Anyone can please help with some advise?

Regards Jan

**Comment by OzGav** [comment, marked as answer]
Not convinced those errors are related to ha-floorplan. Please turn on the ha-floorplan logging https://experiencelovelace.github.io/ha-floorplan/docs/usage/#logging

**Comment by exetico** [comment]
I have not experienced that myself.  What version of HA are you using? I think you're fighting with a non ha-floorplan related issue 😊.

If the problem goes away if you disable ha-floorplan, and shows up again after enabling it again, please remove other 3rd-party solutions, and see if it's still a problem. If so, I'd be happy to hear more about your setup with ha-floorplan, if it's a problem with our solution. I doubt it, but... I'll never say never 😀.

**Comment by jarojetu** [comment]
Hi, thanks both for hints, it was indeed not related to ha-floorplan. I just restarted whole HA, disabled uBlock Origin addon in Edge and it seems to be ok now. Thanks and please close/delete the thread it was irrelevant...
